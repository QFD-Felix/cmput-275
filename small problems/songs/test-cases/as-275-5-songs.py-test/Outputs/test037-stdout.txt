2 16 1 2 5 2 2 6 3 1 14 1 13 2 0 11 0 5 0 0 0 0 8 0 13 2 2 6 1 13 2 0 3 3 0 6 3 3 0 5 18 2 5 5 1 1 3 6 6 14 0 0 0 6 2 0 4 0 7 3 1 6 2 6 11 1 5 3 0 3 3 8 5 1 10 3 0 6 2 3 7 0 3 17 22 3 0 5 2 0 1 4 1 4 0 1 7 2 0 11 1 1 3 1 11 3 10 0 1 4 1 1 2 5 5 2 4 11 16 3 4 0 1 3 8 0 9 0 1 1 1 5 1 5 7 8 1 13 4 0 0 7 2 11 2 0 1 2 3 4 14 12 0 2 3 3 0 3 1 3 5 0 5 3 10 8 2 2 2 3 1 15 3 6 3 1 5 11 0 1 2 18 1 1 1 2 4 2 3 3 16 4 1 1 4 1 8 0 2 10 1 1 1 5 1 11 6 0 3 4 0 11 3 1 3 5 2 8 5 0 8 0 2 6 2 2 5 8 4 1 6 2 5 1 19 12 2 12 2 0 1 1 2 0 6 0 0 14 0 2 4 1 4 4 10 0 4 2 0 4 9 5 0 2 17 2 4 6 0 10 4 0 0 3 4 0 1 6 2 10 6 4 0 5 0 11 1 8 6 2 0 1 5 0 15 3 8 3 4 0 1 4 1 5 2 6 6 0 2 5 5 6 8 0 14 7 0 2 0 7 3 0 2 8 1 5 7 0 4 0 4 4 7 17 3 1 2 3 8 2 5 2 0 1 4 11 7 0 1 12 1 3 0 9 1 5 1 1 1 6 3 3 8 4 2 11 2 0 3 0 2 0 1 0 0 0 0 0
