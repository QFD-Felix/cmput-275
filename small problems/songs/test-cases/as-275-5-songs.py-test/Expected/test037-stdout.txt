2 18 2 0 3 0 5 2 2 6 3 1 14 1 13 2 0 11 0 5 0 0 0 0 8 0 13 2 2 6 1 13 2 0 3 3 0 6 3 3 0 5 7 2 4 7 1 3 5 2 12 6 6 14 0 0 0 6 2 0 4 0 7 3 1 6 2 6 11 1 5 3 0 3 3 8 5 1 10 3 0 6 2 3 7 0 3 17 22 3 0 5 2 0 1 4 1 4 0 1 7 2 0 11 1 1 3 1 11 3 10 0 1 4 1 1 2 5 5 2 4 11 18 3 4 0 1 3 8 0 8 0 1 1 1 2 3 7 4 2 7 8 1 14 4 0 0 8 3 1 3 10 0 1 2 3 4 16 14 0 4 5 0 0 3 0 3 1 3 5 0 5 3 10 8 2 2 2 3 1 15 3 6 3 1 5 11 0 1 2 19 1 1 1 2 4 2 3 3 17 2 2 2 2 4 1 8 0 2 10 1 1 1 5 1 11 6 0 3 4 0 11 3 1 3 5 2 8 5 0 8 0 2 6 2 2 5 8 4 1 6 2 5 1 19 12 2 12 2 0 1 1 2 0 6 0 0 14 0 2 4 1 4 4 10 0 4 2 0 4 9 5 0 2 19 2 4 6 0 12 4 0 0 3 4 0 1 8 1 1 2 10 6 4 0 5 0 11 1 8 6 2 0 1 5 0 15 3 8 3 4 0 1 4 1 5 2 6 6 0 2 5 5 8 4 0 16 2 2 5 6 2 0 7 3 0 2 8 1 5 7 0 4 0 4 4 7 17 3 1 2 3 8 2 5 2 0 1 4 11 7 0 1 12 1 3 0 9 1 5 1 1 1 6 3 3 8 4 2 6 2 0 4 0 3 0 5 1 0 0 0 0 0
